.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,73 @@
+name: 🐞 Bug
+description: Report a bug encountered while using prometheus-operator
+title: "[BUG] <title>"
+labels: ["kind/bug", "needs-triage"]
+body:
+- type: checkboxes
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues
+      required: true
+- type: textarea
+  attributes:
+    label: What happened?
+    description: A concise description of what you're experiencing.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Did you expect to see something different?
+    description: A concise description of what you expected to happen.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Steps To Reproduce (as minimally and precisely as possible)
+    description: Steps to reproduce the behavior.
+    placeholder: |
+      1. In this environment...
+      1. With this config...
+      1. Run '...'
+      1. See error...
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Environment
+    description: |
+      examples:
+        - **Prometheus Operator version**: v0.69.1
+        - **Kubernetes Version**: 1.27
+        - **Kubernetes Cluster Type**: kind
+    value: |
+        - Prometheus Operator version (please provide operator version and not kube-prometheus/helm chart version):
```

Comment:
Is it necessary to ask prometheus-operator startup parameters?

Reply:
yes because it might explain situations where monitoring resources aren't selected as expected.